src/index.css

```diff
@@ -1,27 +1,366 @@
+:root {
+  --logocolor: #0290ff;
+  --textcolor: #121212;
+  --secondcolor: #4386bf;
+  --progresscolor: #e8e8e8;
+}
+
 body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  display: flex;
+  align-items: center;
+  margin: 200px 200px;
+  background-color: #fafafa;
+  padding: 20px;
+  box-shadow: 0 5px 8px 0 rgba(0, 0, 0, 0.2);
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  flex-direction: row;
+  border-bottom: solid 1px #e8e8e8;
+  box-shadow: rgba(149, 157, 165, 0.2) 0 8px 24px;
+}
+
+.nav-items {
+  margin: 29px 30px;
+}
+
+.nav-item {
+  text-decoration: none;
+  color: var(--textcolor);
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1rem 2.563rem 0.688rem 0;
+  font-size: 1.1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+}
+
+#logo {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 1.5rem 2.938rem 0.125rem 8rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--logocolor);
+}
+
+.logo-section {
+  width: 2.813rem;
+  height: 2.813rem;
+  margin: 1.2rem 0 0 27.4rem;
+  padding: 0;
+  border: solid 1px #e8e8e8;
+  border-radius: 30px;
+}
+
+.user-icon {
+  position: relative;
+  top: 0.8rem;
+  left: 0.85rem;
+  color: var(--logocolor);
+}
+
+.single-book {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.book {
+  display: flex;
+  flex-direction: row;
+  margin-bottom: 13px;
+  margin-top: 50px;
+  width: 80%;
+  border: solid 1px #e8e8e8;
+  background-color: white;
+  padding: 10px 0 50px 0;
+}
```

Comment:
- [ ] Dear @Daniel-Kigozi, Your design is looking good already, But you have not added the `Circe` icon next to the `number with percent` to represent the percentage. 
- [ ] Please, notice how elements are aligned inside the [Zeplin](https://zeplin.io/) design, The cards and the `Book Store` text are aligned equally on the left side and the `user icon` and the `books` are aligned equally on the right side. Please consider fixing this for a better layout.
- [ ] Also, Please notice there is a color difference between your design and [Zeplin](https://zeplin.io/) design on the `Book subtitle`.

### Figma design 
![figma](https://user-images.githubusercontent.com/69077061/218084893-4043c055-01d3-41e1-9cb2-fa965a27275a.PNG)

### Your design.
![your](https://user-images.githubusercontent.com/69077061/218084900-8c1b0fb9-36aa-4bcb-8f82-7e09a0022b09.PNG)